src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -30,15 +30,15 @@ private class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgr
                 _definition = definition;
             }
 
-            public async Task OnReferenceFoundAsync(Document document, TextSpan span)
+            public async Task OnReferenceFoundAsync(Document document, TextSpan span, CancellationToken cancellationToken)
             {
                 var documentSpan = await ClassifiedSpansAndHighlightSpan.GetClassifiedDocumentSpanAsync(
                     document, span, _context.CancellationToken).ConfigureAwait(false);
```

Comment:
I have a note here on the side to "remove `IFindUsagesContext.CancellationToken`. This may be the scenario which prompted me to write that.